### DISCUSSION TITLE
[MTG] Topic for the Community Meeting on Jul 10th

### DISCUSSION THREAD

**Discussion by khanhtc1202** [pipe-cd/pipecd#4455] [General]
Place to collect community meeting topic

**Comment by khanhtc1202** [comment]
About introduce pipecd terraform provider

**Comment by khanhtc1202** [comment]
Good first issue event announcement 
https://pipecd.connpass.com/event/288198/

**Comment by kentakozuka** [comment]
What's included in v0.44.2?

**Comment by kentakozuka** [comment]
The meeting is now available on https://www.youtube.com/watch?v=B8NWgzLNe_o
